scripts/better-table-view.js

```diff
@@ -4,11 +4,11 @@ import { dropEventOnTable } from './core/brt-helper.js';
 
 export class BetterRT {
     static async enhanceRollTableView(rollTableConfig, html, rollTable) {
-        const tableClassName = rollTable.cssClass;// "editable";
-        const tableEntity = rollTableConfig.object;
-        const selectedTableType = tableEntity.getFlag(BRTCONFIG.NAMESPACE, BRTCONFIG.TABLE_TYPE_KEY) || BRTCONFIG.TABLE_TYPE_NONE;
+        const tableClassName = rollTable.cssClass,// "editable";
```

Comment:
slightly better for js performance to chain the const declaration/assignment with a comma.